src/main/java/core/basesyntax/Color.java

```diff
@@ -1,11 +1,21 @@
 package core.basesyntax;
 
 public enum Color {
-  white,
-  red,
-  blue,
-  black,
-  orange,
-  purple,
-  green;
+  WHITE("white"),
+  RED("red"),
+  BLUE("blue"),
+  BLACK("black"),
+  ORANGE("orange"),
+  PURPLE("purple"),
+  GREEN("green");
+
+    private final String value;
+
+    Color(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }

```

Comment:
Good point. But you either can use `Color.name().toLowerCase()`, see next comment